frontend/src/resources/utils/getLastLanguage.ts

```diff
@@ -0,0 +1,5 @@
+/* Copyright Contributors to the Open Cluster Management project */
+import { LAST_LANGUAGE_LOCAL_STORAGE_KEY } from './constants'
+
+export const getLastLanguage = (): string => localStorage.getItem(LAST_LANGUAGE_LOCAL_STORAGE_KEY) ?? navigator.language
+console.log('GET LAST LANG : :', getLastLanguage())
```

Comment:
Is this intended?

Reply:
@yiraeChristineKim good catch, will remove that!